app/mailers/patient_mailer.rb

```diff
@@ -263,6 +202,10 @@ def add_fail_history_sms(patient)
     History.report_reminder(patient: patient, comment: comment)
   end
 
+  def add_fail_history_voice(patient)
+    History.report_reminder(patient: patient, comment: "Sara Alert failed to call monitoree at #{patient.primary_telephone}.")
```

Comment:
I think we wanted to avoid language such as "Sara Alert failed" but rather "Sara Alert was unable due to...." - I would check with PO team since this has come up before. I also think this shouldn't be a new separate method but rather just utilize the existing `add_fail_history_sms`, rename it, and allow it to take a parameter to switch out "text" for "call" if need be.